controllers/api/eventRoutes.js

```diff
@@ -18,9 +18,14 @@ router.get('/', withAuth, async (req, res) => {
 
 router.get('/:id', withAuth, async (req, res) => {
     try {
+        const vendorData = await Vendor.findAll();
         const eventData = await Event.findByPk(req.params.id);
-
-        res.status(200).json(eventData);
+        const venueData = await Venue.findAll();
+          const venues = venueData.map((venue) => venue.get({ plain: true }));
+            const event = eventData.get({ plain: true });
+        const vendors = vendorData.map((vendor) => vendor.get({ plain: true }));
+        res.render('events', { vendors, event, venues,
+          logged_in: req.session.logged_in });
```

Comment:
I think you've broken this api route by changing the return from json of event data to a rendered HTML page